gradle/android-dbflow/app/src/main/java/mobi/porquenao/poc/kotlin/ui/MainActivity.kt

```diff
@@ -0,0 +1,37 @@
+package mobi.porquenao.poc.kotlin.ui
+
+import android.os.Bundle
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.Menu
+import android.view.MenuItem
+import mobi.porquenao.poc.kotlin.R
+
+public class MainActivity : BaseActivity() {
+
+    lateinit var mAdapter: MainAdapter
+    lateinit var vList: RecyclerView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.main)
+        vList = findViewById(R.id.list) as RecyclerView
```

Comment:
Android Extensions require the corresponding IDEA plugin to be installed.
Also, IMO, sample projects like this should contain as few external dependencies as possible.
Probably it is better to provide a separate example that takes advantage of Android Extensions plugin.